spec/requests/page_views_spec.rb

```diff
@@ -35,6 +35,15 @@
         }
         expect(PageView.last.user_agent).to eq("test")
       end
+
+      it "sends visited_page_category" do
+        post "/page_views", params: {
+          article_id: article.id,
+          user_agent: "test",
+          visited_page_category: "stories-show"
+        }
+        expect(PageView.last.visited_page_category).to eq("stories-show")
```

Comment:
as a good rule to keep in mind, instead of last it's safer to check the collection tied to the object you're testing. if we ever get to parallel testing "the last of the model" might not be the object you're looking for. In this case `article.reload.page_views.last` I guess